books/spiders/book.py

```diff
@@ -0,0 +1,69 @@
+import scrapy
+from scrapy.http import Response
+
+
+class BookSpider(scrapy.Spider):
+    name = "book"
+    allowed_domains = ["books.toscrape.com"]
+    start_urls = ["https://books.toscrape.com/"]
+
+    def parse(self, response: Response, **kwargs) -> None:
+        for product_url in response.css("h3 > a::attr(href)").getall():
+            product_url = response.urljoin(product_url)
+            yield scrapy.Request(product_url, callback=self.parse_product)
+
+        next_page = response.css("li.next > a::attr(href)").get()
+
+        if next_page:
+            yield response.follow(next_page, callback=self.parse)
```

Comment:
The pagination logic in the `parse` method is correct, using `response.follow` to navigate to the next page. Ensure that the CSS selector `li.next > a::attr(href)` accurately targets the next page link.